src/mpl/include/mpl_trmem.h

```diff
@@ -230,6 +230,89 @@ typedef struct {
   Utility
   M*/
 
+#ifdef HAVE_MM256_STREAM_SI256
+#include <immintrin.h>
+#endif
+#include "string.h"
+
+static inline void MPL_Memcpy_stream(void *dest, const void *src, size_t n)
+{
+#ifdef HAVE_MM256_STREAM_SI256
+    if (n <= 256) {
+        memcpy(dest, src, n);
+        return;
+    }
+
+    char *d = (char *) dest;
+    const char *s = (const char *) src;
+
+    /* Copy the first 63 bits or less (if the address isn't 64-bit aligned) using a regular memcpy to
+     * make the rest faster */
+    if (((uintptr_t) d) & 63) {
```

Comment:
Another example, the code is using `63` as bytes, but your comments is `63 bits`

Reply:
Yep. This comment is just wrong. I'll fix this.